src/app/components/Search/SearchForm.jsx

```diff
@@ -115,18 +117,18 @@ SearchForm.defaultProps = {
 SearchForm.propTypes = {
   searchResults: PropTypes.object,
   query: PropTypes.string,
-  filter: PropTypes.string,
-  allowedFilters: PropTypes.object,
+  field: PropTypes.string,
+  allowedFields: PropTypes.object,
   sort: PropTypes.object,
   search: PropTypes.func,
 };
 
 const mapDispatchToProps = dispatch => ({
-  search: (query, filter) => dispatch(search(query, filter)),
+  search: (query, field) => dispatch(search(query, field)),
 });
 
 
-export default connect(
+export default withRouter(connect(
```

Comment:
Never used `withRouter` but will agree with what you do.

Reply:
Am going to move this to the next PR since I'm prepping for adding changes to server.js, etc. for server-side rendering and routing. It was suggested in the React Router docs to use this when implementing Redux on a project.